packages/runtime/src/lib/txn.ts

```diff
@@ -185,6 +185,13 @@ export function txnSpecByField(
 			break;
 		}
 
+		case "StateProofPK": {
+			// When I run teal debug, "StateProofPK" always return 64 zero bytes.
+			// so I set up 64 zero bytes is default value of StateProofPK
```

Comment:
```suggestion
			// While running teal debugger, "StateProofPK" always return 64 zero bytes.
			// so we set up 64 zero bytes as default value of StateProofPK
```

Reply:
Don't talk in first person, in code comments :sweat_smile:

Reply:
Thanks!